src/main/java/seedu/address/model/person/PersonName.java

```diff
@@ -6,9 +6,10 @@
 public class PersonName {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Person name should only consist of alphabets and digits, and it should not be blank";
+            "Person name must consist of alphabets (may contain digits as well) "
+                    + "and it should not be blank";
 
-    public static final String VALIDATION_REGEX = "^([a-zA-Z])+[a-zA-Z0-9_ ]*";
+    public static final String VALIDATION_REGEX = "^[a-zA-Z0-9_ ]*([a-zA-Z])+[a-zA-Z0-9_ ]*";
```

Comment:
Can I check the rationale for allowing digits in a person's name?

Reply:
You can have names like amy2 if you have 2 friends called amy

Reply:
I don't think it is a very good way to name them like that though. Would be better to include their full name... ? Naming by 1, 2, 3, 4 will be confusing and I don't think thats how people operate, they will just use it with their surname generally.

Reply:
Okay I will remove

Reply:
done

Reply:
also, please try to run the ui once to look for bugs, thanks

Reply:
post merging